tq42/tests/test_utils.py

```diff
@@ -282,3 +282,22 @@ def test_save_get_token_has_NoKeyringError(
         )
         os.remove(token_file_path)
         self.assertEqual(token, "test_token")
+
+    @mock.patch("keyring.get_password")
+    @mock.patch("keyring.set_password")
+    def test_password_set_error(self, mock_set_password, mock_get_password):
```

Comment:
this is test is not asserting that an error is actually getting thrown?
So what is the actual test case? Could we please make it clearer from the naming what we expect?